src/domains/pay-wallet-common/03_cosmosdb.tf

```diff
@@ -155,6 +155,30 @@ module "cosmosdb_pay_wallet_collections" {
   lock_enable = var.env_short != "p" ? false : true
 }
 
+module "cosmosdb_payment_wallets_log_events_collection" {
```

Comment:
do not duplicate this module, use the above `cosmosdb_pay_wallet_collections` one.
You can add a `default_ttl_seconds` parameter for each locals collection parameter valued with null (the [default](https://github.com/pagopa/terraform-azurerm-v3/blob/7dbbc06d591d3ce66536a7bdb2208b1370de04dd/cosmosdb_mongodb_collection/variables.tf#L30-L34) value for the module) and set the 2592000 value for this new collection

Reply:
Together with @infantesimone we valued more important not modifying already existing collections since terraform would consider adding the default ttl an update; we prefer to not have any impact on already existing collections in prod env

Reply:
IDK, I just made the modification locally adding parameter and it gives only diffs in one collection (the one where i set the attribute with a value different from null for test purpose)

Reply:
Considering the module default value is null we can set it explicitly to null for the already existent collections and value it only for the new one. This way the stored terraform state for the already existent collection will remain the same and can configure TTL only where needed (for future collections or when we will set it for the already existent ones 😄)

Reply:
Proposal applied in 8fe27322